preview/snap/snapcraft.yaml

```diff
@@ -0,0 +1,66 @@
+name: powershell-preview
+
+# the version number is not used, but required
+version: 6.0.1
+version-script: |
+  file="./version.txt"
+  if [ -f "$file" ]
+  then
+    cat $file
+  else
+    version=$(curl -s 'https://raw.githubusercontent.com/PowerShell/PowerShell/master/tools/metadata.json' | jq .NextReleaseTag | sed 's/"//g' | sed 's/v//')
+    echo $version
+  fi
+summary: PowerShell for every system
```

Comment:
nit: GitHub has an exclamation mark at the end. Not sure if you want to add that.

Reply:
Added